src/ui/storage/index.ts

```diff
@@ -129,4 +141,40 @@ export default class ApplicationStorage implements IApplicationStorage {
   async setShowPreReleases(value: boolean): Promise<void> {
     localStorage.setItem(UI_SHOW_FIRMWARE_PRE_RELEASES, JSON.stringify(value));
   }
+
+  async setWifiSSID(bindingPhrase: string): Promise<void> {
+    localStorage.setItem(WIFI_SSID_KEY, bindingPhrase);
+  }
+
+  async getWifiSSID(): Promise<string> {
+    const value = localStorage.getItem(WIFI_SSID_KEY);
+    if (value === null) {
+      return '';
+    }
+    return value;
+  }
+
+  async setWifiPassword(bindingPhrase: string): Promise<void> {
+    localStorage.setItem(WIFI_PASSWORD_KEY, bindingPhrase);
+  }
+
+  async getWifiPassword(): Promise<string> {
+    const value = localStorage.getItem(WIFI_PASSWORD_KEY);
+    if (value === null) {
+      return '';
+    }
+    return value;
+  }
+
+  async setRegulatoryDomain900(bindingPhrase: string): Promise<void> {
+    localStorage.setItem(REGULATORY_DOMAIN_900_KEY, bindingPhrase);
+  }
+
+  async getRegulatoryDomain900(): Promise<string> {
```

Comment:
I think it could return UserDefineKey over here instead of doing type conversions later?

Reply:
The get and set functions have been changed to accept and return UserDefineKey instead of string.